DQMOffline/Configuration/python/DQMOffline_SecondStep_cff.py

```diff
@@ -98,7 +98,7 @@
                                          DQMMessageLoggerClientSeq )
 
 
-HLTMonitoringClient = cms.Sequence(trackingMonitorClientHLT * trackingForDisplacedJetMonitorClientHLT)
+HLTMonitoringClient = cms.Sequence(trackingMonitorClientHLT * trackEfficiencyMonitoringClientHLT * trackingForDisplacedJetMonitorClientHLT)
 HLTMonitoringClientPA= cms.Sequence(trackingMonitorClientHLT * PAtrackingMonitorClientHLT)
```

Comment:
Would it make sense to update `HLTMonitoringClientPA` in the same way?

Reply:
It could be useful but, since that's the client used in the Heavy-Ions scenario, I would let @FHead comment

Reply:
@FHead , there is a question on a HLT-DQM sequence for HIon. Do you know who could provide feedback to @kjpena ?

Reply:
Let me tag relevant HI tracking and HLT people @abaty @CesarBernardes @denerslemos to see if they have inputs

Reply:
@abaty @CesarBernardes @denerslemos

Do you have any inputs?

Reply:
Sorry for the delay. Since pA is similar to pp, I think would be good to include trackEfficiencyMonitoringClientHLT in HLTMonitoringClientPA. Thanks

Reply:
Thanks for the input @denerslemos, I will go ahead and update the ```HLTMonitoringClientPA```

Reply:
@denerslemos, please note that HI sequences will not be updated in this PR, as discussed [below](https://github.com/cms-sw/cmssw/pull/38959#discussion_r940564517). If needed, we can follow up with a separate PR

Reply:
@kjpena, ok, thank you so much for the efforts. I think it's a great idea to follow up using a separate PR.